js/apps/admin-ui/src/user-federation/ldap/LdapSettingsSearching.tsx

```diff
@@ -111,227 +109,82 @@ export const LdapSettingsSearching = ({
             )}
           />
         </FormGroup>
-        <FormGroup
+        <TextControl
+          name="config.usersDn.0"
           label={t("usersDN")}
-          labelIcon={
-            <HelpItem helpText={t("usersDNHelp")} fieldLabelId="usersDn" />
-          }
-          fieldId="kc-ui-users-dn"
+          labelIcon={t("usersDNHelp")}
           isRequired
-          validated={
-            (form.formState.errors.config as any)?.usersDn?.[0]
-              ? "error"
-              : "default"
-          }
-          helperTextInvalid={
-            (form.formState.errors.config as any)?.usersDn?.[0].message
-          }
-        >
-          <KeycloakTextInput
-            isRequired
-            defaultValue=""
-            id="kc-ui-users-dn"
-            data-testid="ldap-users-dn"
-            validated={
-              (form.formState.errors.config as any)?.usersDn?.[0]
-                ? "error"
-                : "default"
-            }
-            {...form.register("config.usersDn.0", {
-              required: {
-                value: true,
-                message: t("validateUsersDn").toString(),
-              },
-            })}
-          />
-        </FormGroup>
-        <FormGroup
+          defaultValue=""
```

Comment:
This can be omitted, as it is the default value for `TextControl` if no value was provided.
```suggestion
```